src/style.css

```diff
@@ -1 +1,76 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Add a generic font family as a fallback after 'sans-serif'.